### PR TITLE
Encrypt rds and enable Performance Insights

### PR DESCRIPTION
**Pull request by ghost**
### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/831

### What changes does this PR make to Grapl? Why?
This adds default encryption to RDS and enables Performance Insights

### How were these changes tested?
Ran pulumi and confirmed that everything ran correctly

### CODE REVIEW

**Comment by twunderlich-grapl** [comment]
TODO add a way to pass KMS keys

File: pulumi/infra/postgres.py [hunk 1]
```diff
@@ -140,9 +140,16 @@ def __init__(
...
             password=password.result,
             port=postgres_port,
-            opts=child_opts,
+            # Because we're not specifying a kms key, this will use the default aws/rds key
+            storage_encrypted=True,
+            # Enable performance insights for 7 days, which is free
+            performance_insights_enabled=True,
+            performance_insights_retention_period=7,
+            # We need delete before replace in order to encrypt instances. Once prod is up and running we may want to
+            # remove this option to avoid accidental downtime.
```
Comment (christophermaier) [inline pulumi/infra/postgres.py:153]: Might be good to file an issue for this, just so we don't forget! _That_ would be a fun incident to deal with :joy:

**Review by christophermaier — APPROVED** [review]
👍

**Comment by twunderlich-grapl** [comment]
Re-opening now that compliance is over. Gonna get this merged in

**Review by colin-grapl — APPROVED** [review]

**Pull request merged**